app/src/main/java/com/sun/homecinema/ui/binding/ImageBinding.kt

```diff
@@ -3,10 +3,13 @@ package com.sun.homecinema.ui.binding
 import android.widget.ImageView
 import androidx.databinding.BindingAdapter
 import com.bumptech.glide.Glide
+import com.sun.homecinema.R
+import com.sun.homecinema.utils.ApiConfig
 
 @BindingAdapter("image")
 fun loadImage(view: ImageView, imageUrl: String?) {
     Glide.with(view.context)
-        .load(imageUrl)
+        .load(ApiConfig.getUrlImage(imageUrl))
+        .error(R.drawable.ic_error)
```

Comment:
nên thêm placeholder để view đỡ trống

Reply:
tại cái màn hình của em toàn item là image nếu để placeholder lúc tải nhìn nó xấu quá ạ